src/main/kotlin/grpc/ValidationInterceptor.kt

```diff
@@ -0,0 +1,84 @@
+package se.uulm.snowballr.backend.grpc
+
+import arrow.core.Either
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.grpc.ForwardingServerCallListener.SimpleForwardingServerCallListener
+import io.grpc.Metadata
+import io.grpc.ServerCall
+import io.grpc.ServerCallHandler
+import io.grpc.ServerInterceptor
+import io.grpc.Status
+import se.uulm.snowballr.backend.validation.validateRequest
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * A server interceptor responsible for validating incoming requests in gRPC communication.
```

Comment:
```suggestion
 * A [ServerInterceptor] implementation responsible for validating incoming requests in gRPC communication.
```

Similar to the description in the `LoggingInterceptor` implementation.

Reply:
Good catch! I changed it.